src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -1,154 +1,68 @@
+import { default as useTypedSelector } from 'src/redux/useSelector'
 import colors from 'src/styles/colors'
 import fontStyles from 'src/styles/fonts'
-import { saveProfilePicture } from 'src/utils/image'
-import Logger from 'src/utils/Logger'
-import { Statsig } from 'statsig-react-native'
-import { StatsigDynamicConfigs, StatsigLayers } from 'src/statsig/types'
 
 type Props = NativeStackScreenProps<StackParamList, Screens.NameAndPicture>
-
-const getExperimentParams = () => {
-  try {
-    const statsigLayer = Statsig.getLayer(StatsigLayers.NAME_AND_PICTURE_SCREEN)
-    const showSkipButton = statsigLayer.get(
-      LayerParams[StatsigLayers.NAME_AND_PICTURE_SCREEN].showSkipButton.paramName,
-      LayerParams[StatsigLayers.NAME_AND_PICTURE_SCREEN].showSkipButton.defaultValue
-    )
-    const nameType = statsigLayer.get(
-      LayerParams[StatsigLayers.NAME_AND_PICTURE_SCREEN].nameType.paramName,
-      LayerParams[StatsigLayers.NAME_AND_PICTURE_SCREEN].nameType.defaultValue
-    )
-    return [showSkipButton, nameType]
-  } catch (error) {
-    Logger.warn('NameAndPicture', 'error getting Statsig experiment', error)
-  }
-  return [
-    LayerParams[StatsigLayers.NAME_AND_PICTURE_SCREEN].showSkipButton.defaultValue,
-    LayerParams[StatsigLayers.NAME_AND_PICTURE_SCREEN].nameType.defaultValue,
-  ]
-}
-
-const getBlockedUsernames = (): {
-  blockedAdjectives: string[]
-  blockedNouns: string[]
-} => {
-  try {
-    const config = Statsig.getConfig(StatsigDynamicConfigs.USERNAME_BLOCK_LIST)
-    const blockedAdjectives = config.get(
-      ConfigParams[StatsigDynamicConfigs.USERNAME_BLOCK_LIST].blockedAdjectives.paramName,
-      ConfigParams[StatsigDynamicConfigs.USERNAME_BLOCK_LIST].blockedAdjectives.defaultValue
-    )
-    const blockedNouns = config.get(
-      ConfigParams[StatsigDynamicConfigs.USERNAME_BLOCK_LIST].blockedNouns.paramName,
-      ConfigParams[StatsigDynamicConfigs.USERNAME_BLOCK_LIST].blockedNouns.defaultValue
-    )
-    return { blockedAdjectives, blockedNouns }
-  } catch (error) {
-    Logger.warn('NameAndPicture', 'error getting Statsig blocked usernames', error)
-  }
-  return {
-    blockedAdjectives: [],
-    blockedNouns: [],
-  }
-}
-
 function NameAndPicture({ navigation, route }: Props) {
   const [nameInput, setNameInput] = useState('')
-  const [showSkipButton, nameType] = useMemo(getExperimentParams, [])
-  const { blockedAdjectives, blockedNouns } = useMemo(getBlockedUsernames, [])
   const cachedName = useTypedSelector(nameSelector)
-  const picture = useTypedSelector((state) => state.account.pictureUri)
   const choseToRestoreAccount = useTypedSelector((state) => state.account.choseToRestoreAccount)
   const recoveringFromStoreWipe = useTypedSelector(recoveringFromStoreWipeSelector)
   const { step, totalSteps } = useTypedSelector(registrationStepsSelector)
-  const shouldSkipProfilePicture = useTypedSelector((state) => state.app.skipProfilePicture)
```

Comment:
should we clean up this remote config flag from app state? Guess we could also make a single remote config flag cleanup once all dependencies on other flags (`showGuidedOnboarding`, `createAccountCopyTestType`) are removed.

Reply:
I like the single pr idea!